package.json

```diff
@@ -59,6 +59,8 @@
   "dependencies": {
```

Comment:
Rename commit `Add dependencies: elkjs, web-worker` to `Add dependencies: elkjs, nunjucks, web-worker`

Reply:
I have not added `nunjucks` as a dependency.
See [https://github.com/ditrit/leto-modelizer-plugin-core/blob/main/package.json](https://github.com/ditrit/leto-modelizer-plugin-core/blob/main/package.json)